compiler/tools/fpp-to-cpp/test/array/S3SerializableAc.ref.cpp

```diff
@@ -160,17 +160,33 @@ namespace S {
   {
     static const char* formatString =
       "( "
-      "mU32Array = [ %" PRIu32 ", "
-      "%" PRIu32 ", "
-      "%" PRIu32 " ], "
+      "mU32Array = [ %s ], "
       "mF64 = %f"
       " )";
 
+    // Declare strings to hold any serializable toString() arguments
+    Fw::String mU32ArrayStr;
+
+    // Call toString for arrays and serializable types
+    for (FwSizeType i = 0; i < 3; i++) {
+      Fw::String mU32ArrayTmp;
+      mU32ArrayTmp.format("%" PRIu32 "", this->m_mU32Array[i]);
+
+      FwSizeType size = mU32ArrayTmp.length() + (i > 0 ? 2 : 0);
+      FwSizeType remaining = mU32ArrayStr.maxLength() - mU32ArrayStr.length();
```

Comment:
`FwSizeType` is unsigned, so we should avoid subtraction here, which can overflow. Let's refactor the arithmetic to use addition.